site/docs/ch/meta-service/table-management.md

```diff
@@ -0,0 +1,57 @@
+# 表管理
+[AMS Dashboard](http://localhost:1630)中提供了命令行工具`Terminal`帮组用户快速完成表的创建、修改与删除操作。
+同时还可以在[Spark](#flink.md)和[Flink](#spark/spark-ddl.md)等引擎中使用SQL完成表的管理。
+
+## 新建表
+登录[AMS Dashboard](http://localhost:1630)后，进入`Terminal`，输入建表语句并执行即可完成表的创建。
```

Comment:
Is there a place to show the default username and password?

Reply:
I mentioned this on the `Arctic Dashboard` page.
User DO need deploy the ams before createing table.